ubuntu

```diff
@@ -28,17 +32,17 @@ sudo apt-get update
 sudo apt-get -yq install clang libicu-dev git wget
 
 echo "⬇️  Downloading"
-wget https://swift.org/builds/swift-3.0-release/$OS/swift-3.0-RELEASE/$TC.tar.gz
+wget https://swift.org/builds/swift-3.0.1-release/$OS/swift-3.0.1-RELEASE/$TC.tar.gz
 
 echo "🔧  Untarring"
 tar zxf $TC.tar.gz
 rm -rf $TC.tar.gz
 
 echo "🚀  Installing";
 echo "🔒  Sudo required";
-sudo mv $TC /swift-3.0
+sudo mv $TC /swift-3.0.1
 
-echo "export PATH=/swift-3.0/usr/bin:\"\${PATH}\"" >> ~/.bashrc
-source ~/.bashrc
+echo "export PATH=/swift-3.0.1/usr/bin:\"\${PATH}\"" >> ~/.bashrc
+exec bash
```

Comment:
Here is a good explanation http://wiki.bash-hackers.org/commands/builtin/exec
It replaces the current bash with a new bash without logging the user out.
I had trouble with the `source ~/.bashrc`, I needed to do that manually.
After a investigation I found the `exec bash` command and it's working fine.